dashboards-reports/package.json

```diff
@@ -19,7 +19,7 @@
     "async-mutex": "^0.2.6",
     "babel-polyfill": "^6.26.0",
     "cron-validator": "^1.1.1",
-    "dompurify": "^2.3.8",
+    "dompurify": "^2.4.1",
```

Comment:
Is this change intentional? don't find mention of this in PR description

Reply:
Yes it is intentional. Build was failing with error 
```ERROR [single_version_dependencies] Multiple version ranges for the same dependency
      were found declared across different package.json files. Please consolidate
      those to match across all package.json files. Different versions for the
      same dependency is not supported.

      If you have questions about this please reach out to the operations team.

      The conflicting dependencies are:

        dompurify
          ^2.4.1 => opensearch-dashboards
          ^2.3.8 => reports-dashboards
error Command failed with exit code 1.
```
Ref: https://github.com/opensearch-project/dashboards-reporting/actions/runs/3701723685/jobs/6271294102

Reply:
added in PR description